addons/mass_mailing/views/snippets.xml

```diff
@@ -38,5 +38,85 @@
+                        <div class="alert alert-warning alert-dismissible fade in css_non_editable_mode_hidden o_not_editable" role="alert" contenteditable="false">
+                            <strong>newsletter Popup! </strong>newsletter popup snippet effect is active on this page. Click Here To <a id="edit_dialog" href="#"> Edit Dialog Content</a>
+                        </div>
+                        <div class="modal modal-dialog modal-md fade fade-custom" id="newsletter_popup" tabindex="-1" role="dialog">
+                            <div class="modal-content o_popup_modal_content">
+                                <a href="#" class="o_popup_btn_close o_not_editable" data-dismiss="modal" contenteditable="false">&amp;times;</a>
+
+                                <div class="o_popup_modal_body text-center">
+                                    <div class="o_popup_content_dev" data-oe-placeholder="Type Here ..."/>
+                                    <div class="input-group o_not_editable o_popup_animated o_popup_bounce_del500ms o_popup_bounce_small" contenteditable="false">
+                                        <input type="email" name="email" class="popup_subscribe_email form-control" placeholder="your email..." autofocus="autofocus"/>
+                                        <span class="input-group-btn">
+                                            <a href="#" class="btn btn-success popup_subscribe_btn">Subscribe</a>
+                                        </span>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </xpath>
+        </template>
+
+        <template id="website_popup_options" name="Popup Options" inherit_id="website.snippet_options">
+            <xpath expr="//div" position="after">
+                <div data-js='newsletter_popup'
+                    data-selector=".o_newsletter_popup"
+                    data-drop-in=".oe_structure, [data-oe-type=html]">
+                    <li data-select_mailing_list=""><a>Change Newsletter</a></li>
+                </div>
+            </xpath>
+        </template>
+
+        <template id="FieldTextHtmlPopupContent" name="Editor layout">
+
+            <t t-set="head">
+                <t t-if="enable_editor">
+                    <link rel="stylesheet" href="/website_mail/static/src/css/snippets.css" type="text/css"/>
+                    <link rel="stylesheet" type="text/less" href="/mass_mailing/static/src/css/mass_mailing_popup.css" />
+                    <link rel="stylesheet" href="/website_mail/static/src/css/website_mail_iframe.css" type="text/css"/>
+
+                </t>
+             </t>
+
+            <t t-call="website.editor_layout">
+                <div>
+                    <div class="modal-dialog modal-md show" id="newsletter_popup">
```

Comment:
Please remove this default content. The editor should be fixed and should take the field default value (will be done in another task). You just have to be sure that the right css / js / snippets are loaded.